src/themes/mat/colors.js

```diff
@@ -5,7 +5,7 @@ const colorsToShade= {
 	secondary: '#0096a3',
 	grey: '#7a7a7a',
 	darkGrey: '#4a4a4a',
-	type: '#333333',
+	darkness: '#333333',
```

Comment:
what we are a win in this change?)

Reply:
(Mirroring [this change](https://github.com/dbmedialab/shiny/pull/565/commits/68a0b093b4c401c7d5fce0dc9aa37bfaa5fbfe7c#diff-461ba82a7711956629e733397c17c2a0R64) in the default theme)